lib/generators/manageiq/provider/provider_generator.rb

```diff
@@ -10,13 +10,32 @@ class ProviderGenerator < PluginGenerator
     class_option :dummy, :type => :boolean, :default => false,
                  :desc => "Generate dummy implementations (Default: --no-dummy)"
 
+    class_option :manager_type, :type => :string, :default => "cloud",
+                 :desc => "What type of manager to create (Default: \"cloud\")"
```

Comment:
I'm wondering if there is no default and instead we make it mandatory, and give the list here.

Reply:
That works for me

Reply:
Done